openstack-seeder/pkg/seeder/controller/controller.go

```diff
@@ -266,7 +269,7 @@ func (c *SeederController) seedApply(seed *seederv1.OpenstackSeed) {
 	cmd.Stderr = os.Stderr
 
 	if err = cmd.Start(); err != nil {
-		glog.Errorf("ERROR: could not spawn %s: ", seeder_name, err)
+		glog.Errorf("ERROR: could not spawn %s: %v", seeder_name, err)
```

Comment:
FYI, my linter complained about this clearly wrong statement, so I fixed this while having the file open.